### ISSUE TITLE
Explications

### ISSUE THREAD

**Issue by NicolasJamar** [francoisRoyen/challenge-sass#1]
Super projet !
Tu as utilisé un outil particulier pour t'aider, pourras-tu le montrer aux autres demain jeudi ?

**Comment by francoisRoyen** [comment]
Oui, je peux si tu penses que ça peut leurs apporter quelque chose. Et ça ne me ferait pas de tord pour apprendre à présenter... 😅

Je n’ai pas utilisé d’outil particulier. Juste npm pour « npm run sass » et « npm run prefix ».

Par contre, j’ai suivi le très très bon cours d’Open ClassRooms sur le sujet !
